src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,13 @@
 
 public class Application {
```

Comment:
`private static final int name_of_constant` according to the task description = 3;